src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/ImageSharpImageCacheTenantEvents.cs

```diff
@@ -0,0 +1,85 @@
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Environment.Shell.Removing;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Media.Azure.Services;
+
+internal class ImageSharpImageCacheTenantEvents : ModularTenantEvents
+{
+    private readonly ImageSharpImageCacheOptions _options;
+    private readonly ShellSettings _shellSettings;
+    private readonly ILogger _logger;
+
+    protected readonly IStringLocalizer S;
+
+    public ImageSharpImageCacheTenantEvents(
+        IOptions<ImageSharpImageCacheOptions> options,
+        ShellSettings shellSettings,
+        ILogger<ImageSharpImageCacheTenantEvents> logger,
+        IStringLocalizer<ImageSharpImageCacheTenantEvents> localizer)
+    {
+        _options = options.Value;
+        _shellSettings = shellSettings;
+        _logger = logger;
+        S = localizer;
+    }
+
+    public override async Task ActivatingAsync()
+    {
+        // Only create container if options are valid.
+        if (_shellSettings.IsUninitialized() ||
```

Comment:
I like to place the operator at the beginning of the line this way commenting out the line will be with the change of 1 line instead of 2 lines.

Reply:
This doesn't actually need to be multiple lines. I just copied the Media Storage implementation.